Source/Kesmai.Server/Game/Entities/Mobiles/Creatures/Undead/Banshee.cs

```diff
@@ -6,25 +6,41 @@ namespace Kesmai.Server.Game
 {
 	public partial class Banshee : CreatureEntity, IUndead
 	{
+
+		public override CreatureImmunity Immunity { get; set; } = CreatureImmunity.Piercing;
```

Comment:
pretty sure this won't compile.  I think you need to do public override CreatureImmunity Imunity { get; set; } = CreatureImunity.Piercing || Creature Immunity.Slashing etc.

Reply:
![image](https://user-images.githubusercontent.com/86573170/180118189-0d139555-37d7-4749-9973-92dc9a998f3a.png)

Swordmaster is a pre-existing entity that Lux made so I think it would actually work.

Reply:
not sure where you are seeing that code, but it doesn't look like that to me in master.

Reply:
I added the other immunities I get what youre saying now I'll fix it so the pull is nice and ready